core/templates/dev/head/pages/admin-page/misc-tab/admin-misc-tab.directive.ts

```diff
@@ -142,6 +144,17 @@ angular.module('oppia').directive('adminMiscTab', [
           ctrl.dataExtractionQueryStatusMessage = message;
         };
 
+
+        ctrl.sendDummyMailToAdmin = function() {
+          $http.post(SEND_DUMMY_MAIL_HANDLER_URL)
+            .then(function(response) {
+              ctrl.setStatusMessage(response.data.msg);
+            }, function(errorResponse) {
+              ctrl.setStatusMessage('Server error: ' +
```

Comment:
When the contents of a paren span multiple lines, break after the opening paren.

Reply:
Done.